### ISSUE TITLE
Development activities for A11y Feedback Mechanism

### ISSUE THREAD

**Issue by connorscarolyns** [cds-snc/notification-planning#2079]
[New bilingual content in Figma ](https://www.figma.com/design/f6QILuxf94EHaStf2ejWJb/Contact-us-%7C-Nous-joindre?node-id=1781-1303&p=f&focus-id=2120-2546&m=dev)
Existing screens have the changes shown in pink. New screens are all in black.

**Comment by schancase** [comment]
moved to 'plan to finish by' sprint 16 pipeline in quarterly view (as a hunch). cc @connorscarolyns

**Comment by schancase** [comment]
team standup - decided this doesn't need bug bash before going live.

**Comment by connorscarolyns** [comment]
all complete
